src/components/Header/Header.tsx

```diff
@@ -1,68 +1,111 @@
-          >
-            <Image src={DATA.logo} alt="" width={30} height={30} className="bi me-2" />
-          </a>
+        <div className="logo">
+          <Link href = {"/"} legacyBehavior>
+            <a onClick = { () => setActiveIdx(0) }>
+              <Image src = { logo } alt="logo" />
+            </a>
+          </Link>
+        </div>
 
-          <ul className="nav nav-pills">
-            <li className="nav-item">
-              <a href="#top" className="nav-link px-2 text-dark">
-                Home
-              </a>
-            </li>
-            <li className="nav-item">
-              <a href="#top" className="nav-link px-2 text-dark">
-                About Us
-              </a>
-            </li>
-            <li className="nav-item">
-              <a href="#top" className="nav-link px-2 text-dark">
-                News
-              </a>
-            </li>
-            <li className="nav-item">
-              <a href="#top" className="nav-link px-2 text-dark">
-                Volunteer
-              </a>
-            </li>
-          </ul>
+        <nav className="nav">
 
-          <div
-            id="buttons"
-            className="d-flex justify-content-center align-items-center gap-2"
-          >
-            <input
-              type="button"
-              value="Get Help"
-              className="btn btn-outline-dark"
-            />
-            <input type="button" value="Volunteer" className="btn btn-dark" />
+          <div className = "nav__menu-bar" onClick = { () => setNavActive(!navActive) }>
+            <div></div>
+            <div></div>
+            <div></div>
           </div>
 
-          <div className="d-flex align-items-center justify-content-center " id="wrap-toggle">
-            <input type="checkbox" id="switcher" />
-            <label htmlFor="switcher" data-checked="EN" data-unchecked="UKY" className="check"></label>
+          <div className = { `${navActive ? "active" : ""} nav__menu-list` }>
+            {
+              Menu_List.map((menu, idx) => {
+                return(
+                  <div key = {menu.texto} onClick = { () => {
+                    setActiveIdx(idx);
+                    setNavActive(false);
+                  }}>
+                    <Navitem active = { activeIdx === idx } { ...menu } />
+                  </div>
+                )
+              })
+            }
           </div>
 
-        </div>
+          <div className="buttons">
+            <input type="button" value="Get Help" className="btn_help" />
+            <input type="button" value="VOLUNTEER" className="btn_volunteer"/>
+            <div className="wrap-toggle">
+              <input type="checkbox" id="switcher" />
+              <label htmlFor="switcher" data-checked="EN" data-unchecked="UKY" className="check"></label>
+            </div>
+          </div>
+          
+        </nav>
       </div>
+      {/* <nav className="nav">
```

Comment:
Comment section requires careful review.
If it is useful, implement it
If it isn't, delete it